icu[core]:x64-linux build failure

**Host Environment**
- OS: Ubuntu 18.04


**To Reproduce**

`./vcpkg install icu`

**Failure logs**
 
```
-- pkg-config call failed with error code:1
-- pkg-config output:
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:76 (message):
  pkg-config error output:Package 'harfbuzz' requires 'glib-2.0 >= 2.19.1'
  but version of glib-2.0 is
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:297 (vcpkg_fixup_pkgconfig_check_files)
  ports/icu/portfile.cmake:236 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:79 (include)
```

**Additionally**

```
$ ./vcpkg list | grep glib
glib:x64-linux                                     2.52.3-14-7      Portable, general-purpose utility library.
glibmm:x64-linux                                   2.52.1-11        This is glibmm, a C++ API for parts of glib that...
harfbuzz[glib]:x64-linux                                            Glib Unicode callbacks support
```

Had the same error a few days ago, tried `git pull && ./boostrap-vcpkg.sh && ./vcpkg update` and now it's a different error: 

```
$ git rev-parse HEAD
76a7e9248fb3c57350b559966dcaa2d52a5e4458
$ ./vcpkg install icu
Computing installation plan...
The following packages will be built and installed:
    icu[core]:x64-linux
Detecting compiler hash for triplet x64-linux...
Starting package 1/1: icu:x64-linux
Building package icu[core]:x64-linux...
Could not locate cached archive: /home/marco/.cache/vcpkg/archives/8a/8a16084b528de621f192dc810a36da568a039788.zip
-- Using cached /home/marco/vcpkg/downloads/icu4c-67_1-src.tgz
-- Cleaning sources at /home/marco/vcpkg/buildtrees/icu/src/c-67_1-src-39ba50b26e.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/marco/vcpkg/downloads/icu4c-67_1-src.tgz
-- Applying patch /home/marco/vcpkg/ports/icu/disable-escapestr-tool.patch
-- Applying patch /home/marco/vcpkg/ports/icu/remove-MD-from-configure.patch
-- Applying patch /home/marco/vcpkg/ports/icu/fix_parallel_build_on_windows.patch
-- Applying patch /home/marco/vcpkg/ports/icu/fix-extra.patch
-- Using source at /home/marco/vcpkg/buildtrees/icu/src/c-67_1-src-39ba50b26e.clean
-- Configuring x64-linux-rel
-- Configuring x64-linux-rel done
-- Configuring x64-linux-dbg
-- Configuring x64-linux-dbg done
-- Package x64-linux-rel
-- Package x64-linux-rel done
-- Package x64-linux-dbg
-- Package x64-linux-dbg done
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/derb
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/genbrk
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/genccode
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/gencfu
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/gencmn
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/gencnval
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/gendict
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/gennorm2
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/genrb
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/gensprep
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/icupkg
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/makeconv
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/pkgdata
-- Installing: /home/marco/vcpkg/packages/icu_x64-linux/tools/icu/uconv
-- Fixing pkgconfig file: /home/marco/vcpkg/packages/icu_x64-linux/lib/pkgconfig/icu-i18n.pc
-- Fixing pkgconfig file: /home/marco/vcpkg/packages/icu_x64-linux/lib/pkgconfig/icu-io.pc
-- Fixing pkgconfig file: /home/marco/vcpkg/packages/icu_x64-linux/lib/pkgconfig/icu-lx.pc
-- Fixing pkgconfig file: /home/marco/vcpkg/packages/icu_x64-linux/lib/pkgconfig/icu-uc.pc
-- CHECK_LIB_icu-le-hb_RELEASE:CHECK_LIB_icu-le-hb_RELEASE-NOTFOUND
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:232 (message):
  Library "icu-le-hb" was not found! If it is a system library use the
  SYSTEM_LIBRARIES parameter for the vcpkg_fixup_pkgconfig call! Otherwise,
  correct the *.pc file
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:297 (vcpkg_fixup_pkgconfig_check_files)
  ports/icu/portfile.cmake:236 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:79 (include)


Error: Building package icu:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: icu:x64-linux
  Vcpkg version: 2020.06.15-unknownhash

```

--- a/ports/icu/CONTROL
+++ b/ports/icu/CONTROL
@@ -1,6 +1,6 @@
 Source: icu
 Version: 67.1
-Port-Version: 4
+Port-Version: 5
 Homepage: http://icu-project.org/apiref/icu4c/
 Description: Mature and widely used Unicode and localization library.
 Supports: !(arm|uwp)

--- a/ports/icu/portfile.cmake
+++ b/ports/icu/portfile.cmake
@@ -25,7 +25,7 @@ vcpkg_extract_source_archive_ex(
 vcpkg_find_acquire_program(PYTHON3)
 set(ENV{PYTHON} "${PYTHON3}")
 
-set(CONFIGURE_OPTIONS "--disable-samples --disable-tests")
+set(CONFIGURE_OPTIONS "--disable-samples --disable-tests --disable-layoutex")
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
     set(CONFIGURE_OPTIONS "${CONFIGURE_OPTIONS} --disable-static --enable-shared")

